environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/azure/AzureEnvironmentParameters.java

```diff
@@ -14,6 +14,10 @@
     @ApiModelProperty(EnvironmentModelDescription.RESOURCE_GROUP_PARAMETERS)
     private AzureResourceGroup resourceGroup;
 
+    @Valid
+    @ApiModelProperty(EnvironmentModelDescription.RESOURCE_ENCRYPTION_PARAMETERS)
+    private AzureResourceEncryptionParameters azureResourceEncryptionParameters;
```

Comment:
The inclusion of `azure` seems a bit redundant. How about simply calling the property `resourceEncryptionParameters`? The class name can be left as is, though.